Documentation/MigrateToComposer/MigrationSteps.rst

```diff
@@ -122,7 +122,39 @@ You already know the TER and always used it to install extensions? Fine.
 But with composer, the **preferred way** is to install extensions
 directly from `packagist.org <https://packagist.org>`__. This works great, when the maintainer uploaded them
 to there. Many well known extensions are already available.
-You only need to known the package name. There are multiple ways to find it.
+You only need to known the package name. There are multiple ways to find it:
+
+Notice on extension's TER page
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Extension maintainers optionally can link their TYPO3 extension in TER with the
+correct composer key on `packagist.org <https://packagist.org>`__. Some maintainers already did that and if you
+search the extension in TER, you will see a message, which command and composer
+key you can use to install this extension.
+
+|TER composer command|
+
+.. note ::
+
+    The command ``composer req`` is short for ``composer require``. Both commands
+    exactly do the same and are interchangeable.
+
+Check in TER satis
+~~~~~~~~~~~~~~~~~~
+
+If you search the extension in https://composer.typo3.org/satis.html and it's linked to
+`packagist.org <https://packagist.org>`__, they are marked as "abandoned" and you
+will see a message, which composer key should be used to install this extension.
+
+|satis abandoned note|
+
+See warning during composer require command
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you still install one of the abandoned extensions via its ``typo3-ter`` package key,
```

Comment:
You can use single backticks here, see https://docs.typo3.org/typo3cms/HowToDocument/WritingReST/InlineCode.html#preferred-use-single-backticks